# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/wasi-libc/hotfix-no-check.patch
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- wasi-libc-wasi-sdk-17/Makefile.vanilla	2023-03-19 13:45:03.122575699 +0100
+++ wasi-libc-wasi-sdk-17/Makefile	2023-03-19 13:46:04.199576361 +0100
@@ -591,7 +591,7 @@
 # alloctor (providing malloc, calloc, free, etc). Skip this step if the build
 # is done without a malloc implementation.
 ifneq ($(MALLOC_IMPL),none)
-finish: check-symbols
+finish: # check-symbols
 endif
 
 DEFINED_SYMBOLS = $(SYSROOT_SHARE)/defined-symbols.txt
